### DISCUSSION TITLE
Domain name exists

### DISCUSSION THREAD

**Discussion by devgalib** [vercel/community#3842] [Help]
### Summary

Hi, I deleted my project deployed on Vercel (imported from the GitHub Pages repository) and removed all custom domain settings, but I still intermittently received the following message when I accessed the domain, 

### Steps to Reproduce

404: NOT_FOUND
Code: DEPLOYMENT_NOT_FOUND
ID: bom1::9ph72-1693416827213-7745984423ef

[This deployment cannot be found. For more information and troubleshooting, see our documentation.](https://vercel.com/docs/errors/platform-error-codes#deployment_not_found)

**Comment by amyegan** [comment]
Hey @codergalib2005. Did you point the domain at a different server, yet? Or is it still pointed at Vercel?
